### PR TITLE
FI master -> xplat

### PR DESCRIPTION
**Pull request by rainersigwald**

### CODE REVIEW

File: src/Shared/UnitTests/ObjectModelHelpers.cs [hunk 1]
```diff
@@ -1620,5 +1610,86 @@ private static TempPaths GetTempPaths()
                 return tempPaths;
             }
         }
+
+        internal class TestProjectWithFiles : IDisposable
```
Comment (rainersigwald) [inline src/Shared/UnitTests/ObjectModelHelpers.cs]: @cdmihai this conflicted and I took both this class and the one above.

File: ref/netstandard1.3/Microsoft.Build/Microsoft.Build.cs [hunk 2]
```diff
@@ -536,6 +536,7 @@ public partial class Project
         public bool SkipEvaluation { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
         public string SubToolsetVersion { get { throw null; } }
         public System.Collections.Generic.IDictionary<string, Microsoft.Build.Execution.ProjectTargetInstance> Targets { [System.Diagnostics.DebuggerStepThroughAttribute]get { throw null; } }
+        public bool ThrowInsteadOfSplittingItemElement { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
```
Comment (rainersigwald) [inline ref/netstandard1.3/Microsoft.Build/Microsoft.Build.cs]: @cdmihai I assume this is an intentional addition.

**Comment by rainersigwald** [comment]
@dotnet-bot test OSX Build for CoreCLR please

File: ref/netstandard1.3/Microsoft.Build/Microsoft.Build.cs (hunk 2 above)
Comment (cdmihai) [inline ref/netstandard1.3/Microsoft.Build/Microsoft.Build.cs]: yes

**Pull request merged**